### ISSUE TITLE
Fix linting in workspace

### ISSUE THREAD

**Issue by bensleveritt** [garden-co/classic-jazz#702]
There are 1.5k linting errors raised by biome. Before we can turn on linting enforcement, we need to reduce this number to 0 somehow.

**Comment by gdorsi** [comment]
Duplicate of https://github.com/garden-co/jazz/issues/939
